StreamChat.xcodeproj/project.pbxproj

```diff
@@ -539,7 +539,7 @@
 		847E946E269C687300E31D0C /* EventsController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 847E946D269C687300E31D0C /* EventsController.swift */; };
 		847F3CEA2689FDEB00D240E0 /* ChatMessageCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 847F3CE92689FDEB00D240E0 /* ChatMessageCell.swift */; };
 		8486CAF926FA51EE00A9AD96 /* EventDTOConverterMiddleware_Tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8486CAF826FA51EE00A9AD96 /* EventDTOConverterMiddleware_Tests.swift */; };
-		848EE3F8277628CB00B21ECA /* User_Tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 848EE3F7277628CB00B21ECA /* User_Tests.swift */; };
+		848EE3F8277628CB00B21ECA /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
Why did this happen?

Reply:
Nice catch @b-onc!
I feel like this is outcome of the rebase against the develop. Ill remove the dead reference, the `User_Tests.swit` doesn't exist on latest develop.